examples/external_request/example1.rb

```diff
@@ -0,0 +1,64 @@
+require 'net/http'
+require 'bundler'
+Bundler.require
+
+require "new_relic/telemetry_sdk"
+
+unless ENV["API_KEY"]
+  raise "No License key supplied.  Export API_KEY environment variable!" 
```

Comment:
We should use the term "API key" in this error message rather than "license key" - they are two different things, so we don't want to cause confusion.

In fact, there are several kinds of API keys for New Relic, so for maximum clarity we will want to note that this one is the [Insert API Key](https://docs.newrelic.com/docs/apis/get-started/intro-apis/types-new-relic-api-keys#event-insert-key).